.golangci.yml

```diff
@@ -13,16 +13,16 @@
 #
 
 run:
-  concurrency: 4
-  timeout: 5m
-  issues-exit-code: 0 # TODO: once all enabled linters are satisfied  
+  concurrency: 8
+  timeout: 2m
   tests: false
```

Comment:
Tests should be linted. Otherwise, the `golangci-lint` coverage is less complete than `go vet`'s.